charts/emissary-ingress/templates/rbac.yaml

```diff
@@ -110,6 +113,10 @@ rules:
     resources: [ "ingresses/status" ]
     verbs: ["update"]
 
+  - apiGroups: [ "apiextensions.k8s.io" ]
+    resources: [ "customresourcedefinitions", '*' ]
+    verbs: [ "list", "update", "patch" ]
```

Comment:
This is in the `Role`/`ClusterRole` for namespaced resources that it has permission to read cluster-wide (unless "single-namespace" is turned on). It is normally a `ClusterRole`, but gets downgraded to a `Role` if single-namespace is turned on.

Because CRDs are non-namespaced resources, they need to go in a `ClusterRole` *even if* single-namespace is turned on; there's an appropriate `ClusterRole` further up in the file.